src/filters/FullPoseEstimator.cpp

```diff
@@ -3,16 +3,17 @@
 #include "../Util.h"
 
 #include <functional>
+#include "../../src/navtypes.h"
```

Comment:
Can you make this `../navtypes.h`? The extra `../src` in the middle isn't necessary.